grandine/src/grandine_config.rs

```diff
@@ -110,7 +109,7 @@ impl GrandineConfig {
             storage_config.eth1_db_size.to_string_as(true),
         );
 
-        info!("Eth1 RPC URLs: [{}]", eth1_rpc_urls.iter().format(", "));
+        info!("Eth1 RPC URLs: {:?}", eth1_rpc_urls);
```

Comment:
This change is required when switching from the `log` to the `tracing` crate, because otherwise a panic ensues 
```
thread 'main' panicked at /home/zarathustra/.cargo/registry/src/index.crates.io-6f17d22bba15001f/itertools-0.13.0/src/format.rs:95:21:
Format: was already formatted once
```